models/iaf_psc_alpha.cpp

```diff
@@ -370,7 +374,15 @@ iaf_psc_alpha::update( Time const& origin, const long from, const long to )
     }
 
     // set new input current
-    S_.y0_ = B_.currents_.get_value( lag );
+    S_.y0_ = new_input[ Buffers_::I0 ];
+
+    // reset input in ring buffer at position lag
+    // note: delegating the reset to a function in the MultiValueRingBuffer
+    // would entail additional costs
+    for ( auto it = new_input.begin(); it < new_input.end(); ++it )
```

Comment:
If we made this an inline function, there should be no call overhead and we avoid code duplication.

Reply:
Done.

Reply:
Also made other frequently called member functions inline.